elementary/utils/time.py

```diff
@@ -83,3 +95,28 @@ def convert_partial_iso_format_to_full_iso_format(partial_iso_format_time: str)
             f'Failed to covert time string: "{partial_iso_format_time}" to ISO format'
         )
         return partial_iso_format_time
+
+
+def get_formatted_timedelta(time_ago_in_s: float) -> str:
```

Comment:
Just for later use - does it work with int as well?
If it does I think we should add it in the typing as mypy can start scream on other users that will use it due to it 🤔

Reply:
Can we add tests for that?